quality_control_stock/models/stock_picking.py

```diff
@@ -36,6 +24,27 @@ def _compute_count_inspections(self):
     failed_inspections = fields.Integer(
         compute="_compute_count_inspections", string="Inspections failed")
 
+    @api.depends('qc_inspections_ids', 'qc_inspections_ids.state')
+    def _compute_count_inspections(self):
+        Inspection = self.env['qc.inspection']
+        inverse_field = 'picking_id'
```

Comment:
Why putting this field name as variable and not `state`. It's used the same number of times... Any way, I prefer in this case to directly use the field name, which is more readable and even shorter in this case.

Reply:
Easier copy and past function to another model. If you said that it hardly understand, I`ll fix it.